tests/js/client/server_parameters/www-authenticate.js

```diff
@@ -35,36 +32,83 @@ if (getOptions === true) {
   };
 }
 const jsunity = require('jsunity');
-const request = require('@arangodb/request').request;
 const crypto = require('@arangodb/crypto');
+const protocols = ["tcp", "h2"];
+const users = require("@arangodb/users");
+const db = require('internal').db;
+let maintainerMode = require('internal').db._version(true)['details']['maintainer-mode'];
+const helper = require('@arangodb/testutils/user-helper');
+const user = "testUser";
+
+let connectWith = function(protocol, user, password) {
+  let endpoint = arango.getEndpoint().replace(/^[a-zA-Z0-9\+]+:/, protocol + ':');
+  arango.reconnect(endpoint, db._name(), user, password);
+};
+
+function HttpAuthenticateSuite() {
 
-function testSuite() {
   return {
-    testUnauthorized : function() {
-      let result = request({ url: "/_api/version", method: "get" });
-      assertEqual(401, result.status);
-      assertTrue(result.headers.hasOwnProperty('www-authenticate'));
+
+    setUp: function() {
+      users.save(user, "");
+      users.grantDatabase(user, '_system', 'rw');
+      helper.switchUser(user);
+    },
+
+    tearDown: function() {
+      helper.switchUser("root");
+      users.remove(user);
+      arango.reconnectWithNewPassword("");
```

Comment:
Can't this be simplified to just
```suggestion
      connectWith("tcp", "root", "");
      users.remove(user);
```
?